allauth/utils.py

```diff
@@ -168,7 +168,8 @@ def serialize_instance(instance):
         try:
             field = instance._meta.get_field(k)
             if isinstance(field, BinaryField):
-                v = force_str(base64.b64encode(v))
+                if v:
```

Comment:
```suggestion
                if v is not None:
```

Otherwise, `b""` would be turned into `None`?

Reply:
I see. Good catch!